models/vista3d/docs/README.md

```diff
@@ -4,7 +4,7 @@ own datasets. To reproduce the paper results, please refer to https://github.com
 
 # Installation Guide
 ```
-pip install "monai[fire]"
+pip install "monai[fire,ignite]" requests huggingface_hub
```

Comment:
These added requirements can be mentioned in `required_packages_version` in the metadata file as well.